api/uninstall/uninstall.go

```diff
@@ -0,0 +1,105 @@
+package uninstall
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"net/http"
+	"time"
+
+	"github.com/gojekfarm/albatross/pkg/helmcli/flags"
+	"github.com/gojekfarm/albatross/pkg/logger"
+
+	"helm.sh/helm/v3/pkg/action"
+	"helm.sh/helm/v3/pkg/release"
+	"helm.sh/helm/v3/pkg/storage/driver"
+)
+
+var errInvalidReleaseName = errors.New("uninstall: invalid release name")
+
+// Request encapsulates an Http Request.
+type Request struct {
+	ReleaseName  string `json:"release_name"`
+	Dryrun       bool   `json:"dry_run,omitempty"`
+	KeepHistory  bool   `json:"keep_history"`
+	DisableHooks bool   `json:"disable_hooks"`
```

Comment:
Ideally omitEmpty tag can be skipped from DryRun as well, since the tag is used while encoding the data back to json and a request object will never be encoded to json, am I correct ??

Reply:
Yes, and you can actually get rid of that for DryRun.